src/GeoExt/selection/FeatureModel.js

```diff
@@ -359,12 +359,14 @@ Ext.define('GeoExt.selection.FeatureModel', {
     recenterToSelectionExtent: function() {
         var map = this.selectControl.map;
         var selectionExtent = this.getSelectionExtent();
-        var selectionExtentZoom = map.getZoomForExtent(selectionExtent, false);
-        if (selectionExtentZoom > map.getZoom()) {
-            map.setCenter(selectionExtent.getCenterLonLat());
-        }
-        else {
-            map.zoomToExtent(selectionExtent);
+        if (selectionExtent !== null) {
```

Comment:
Do you mind changing this to an early `return` statement in the `if`-block? This would help to reduce the level of indentation and emphasize the edge case you have found?